back/app/services/invites/service.rb

```diff
@@ -175,16 +176,18 @@ def save_invitees(invitees)
       end
     end
 
-    invitees.each do |invitee|
-      if @run_side_fx
-        if invitee.previously_new_record?
-          SideFxUserService.new.after_create(invitee, @inviter)
-          SideFxInviteService.new.after_create(invitee.invitee_invite, @inviter)
-        else
-          SideFxUserService.new.after_update(invitee, @inviter)
+    invitees
+      .sort! { |i| i.roles_previously_was.any? { |r| r['type'] == 'admin' } && i.roles.none? { |r| r['type'] == 'admin' } ? 1 : 0 }
```

Comment:
Why did you use `sort!` and not `sort`? 

In some places of this class, the order of invitees is important (e.g., in `validate_invitee`). Later, it can be important in other places too, so maybe using `sort` instead of `sort!` could be safer.

Reply:
I suppose this piece of code in combination with how we increment additional seats can be pretty brittle, so maybe tests are crucial here. If you don't have time now, maybe it makes sense to add a ticket to add tests for it?